src/hip_memory.cpp

```diff
@@ -1367,6 +1619,15 @@ hipError_t hipMemcpy3DAsync(const struct hipMemcpy3DParms* p, hipStream_t stream
     return ihipLogStatus(e);
 }
 
+hipError_t hipExtMemcpyWithStream(void* dst, const void* src, size_t sizeBytes,
```

Comment:
Is this still needed?

Reply:
the new API should be called hipMemcpyWithStream and should have AMD and CUDA implementations.

Reply:
Good eyes gents, that's just an accidental leftover though:)